test/Tests/src/Data/Ordering_Spec.enso

```diff
@@ -42,24 +39,17 @@ type Parent
 # === The Tests ===
 
 spec =
-    Test.group "Default ordered comparator" <|
-        Test.specify "should support custom ordered comparator" <|
+    Test.group "Default comparator" <|
+        Test.specify "should support custom comparator" <|
             Ordering.compare (Ord.Value 1) (Ord.Value 2) . should_equal Ordering.Less
             Ordering.compare (Ord.Value 1) (Ord.Value 1) . should_equal Ordering.Equal
             Ordering.compare (Ord.Value 20) (Ord.Value 1) . should_equal Ordering.Greater
+            Ordering.compare (UPair.Value 1 2) (UPair.Value 2 1) . should_equal Ordering.Equal
 
-        Test.specify "should support equality for custom ordered comparators in atom field" <|
+        Test.specify "should support equality for custom comparators in atom field" <|
             ((Parent.Value (Ord.Value 1)) == (Parent.Value (Ord.Value 1))) . should_be_true
             ((Parent.Value (Ord.Value 1)) == (Parent.Value (Ord.Value 22))) . should_be_false
 
-        Test.specify "should throw Incomparable_Values when comparing types with unordered comparator" <|
```

Comment:
While we don't throw `Incomparable_Values` we could still have the test with expectations?

Reply:
It is already covered by other tests. Now, `Ordering.compare val1 val2` returns `Ordering.Equal` which is fine.